src/main/kotlin/no/nav/helse/flex/sykepengesoknad/SykepengesoknadLagring.kt

```diff
@@ -0,0 +1,56 @@
+package no.nav.helse.flex.sykepengesoknad
+
+import no.nav.helse.flex.logger
+import no.nav.helse.flex.sykepengesoknad.kafka.SoknadsstatusDTO
+import no.nav.helse.flex.sykepengesoknad.kafka.SykepengesoknadDTO
+import org.springframework.stereotype.Component
+import java.time.Instant
+import java.time.LocalDateTime
+import java.time.ZoneId
+
+@Component
+class SykepengesoknadLagring(
+    private val sykepengesoknadRepository: SykepengesoknadRepository,
+) {
+    val log = logger()
+
+    fun handterSoknad(soknad: SykepengesoknadDTO) {
+        if (soknad.status === SoknadsstatusDTO.SENDT) {
```

Comment:
Denne burde kanskje være '=='  (sjekk på equals())i stedet for '===' (peker på samme objekt)?